packages/driver/src/cypress.js

```diff
@@ -596,6 +598,21 @@ class $Cypress {
     return throwPrivateCommandInterface('addUtilityCommand')
   }
 
+  get currentTest () {
+    const r = this.cy.state('runnable')
+
+    if (!r) {
+      const invocationStack = $stackUtils.getInvocationDetails(this.cy.state('specWindow'), this.config)?.stack
+
+      throw errByPath(errs.currentTest.outside_test)
```

Comment:
Why augment the `erryByPath` api instead of calling `errByPath('currentTest.outside_test')`?

Reply:
Also, there's `$errUtils.throwErrByPath`

Reply:
i guess it does add some runtime overhead, it was to satisfy intellisense. so no reason

Reply:
addressed